src/Extractor/JsonDecoder.php

```diff
@@ -56,7 +56,7 @@ protected function parse(): Generator
 
             // Decode JSON document, throw the JsonException on the error
             if (trim($json) !== '') {
-                yield json_decode($json, true, 512, JSON_THROW_ON_ERROR);
+                yield json_decode($json, false, 512, JSON_THROW_ON_ERROR);
```

Comment:
Zistil som, ze potrebujem objekt a nie pole (v niektorych specifickych situaciach to nebolo `csvmap` kniznici jedno).